docs/tutorials/getting_data.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>1) margin caches - add pointer to margin cache topic notebook</p><p>2) pre-select - also say that you can see the schema later</p><p> </p><p>or switch to a flow like:</p><p> </p><p>gaia_dr3 = read_hipscat(..., WITHOUT the columns arg)</p><p>gaia_dr3.schema</p><p> </p><p>Let's just use the columns we need instead:</p><p> </p><p>gaia_dr3 = read_hipscat(..., WITH the columns arg)</p><p>gaia_dr3.schema</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks. I just changed the flow, it feels closer to what users would experience!